mixins/localize-mixin.js

```diff
@@ -128,6 +143,29 @@ export const LocalizeMixin = dedupeMixin(superclass => class extends superclass
 
 	}
 
+	localizeHTML(key, {
+		link = '', tooltipHelp = '', _html = '',
+		links = {}, tooltipHelps = {},
```

Comment:
1. Can't say I love it myself, but I'm less a fan of allowing and maintaining multiple types in one argument. We could simply say it has to be named and provide an object, but since probably 99% of cases are going to use a single link, I wanted to avoid that.
2. Yeah, I'd not be opposed to using `[link]` to match
3. The text should definitely be run through `this.localize` _almost_ every time, but that's the consumer's responsibility. Maybe it's user-provided. Maybe it's server-generated.

As for providing the link attributes, I agree it's not super pretty. It's just super simple. There was previous discussion [here](https://github.com/BrightspaceUI/core/pull/3211#discussion_r1089089517).